ProfileService.js

```diff
@@ -45,12 +36,17 @@ export class ProfileService {
   async create({
     url = this.config.urls.base, account, didMethod, didOptions} = {}) {
     try {
-      const response = await this._axios.post(
-        url, {account, didMethod, didOptions}
-      );
+      if(url.startsWith('/')) {
+        url = url.substring(1);
+      }
+      const response = await httpClient.post(
+        url, {
+          prefixUrl: this.baseURL,
+          json: {account, didMethod, didOptions}
+        });
       return response.data;
     } catch(e) {
-      _rethrowAxiosError(e);
+      _rethrowHttpError(e);
```

Comment:
This function should be removed in all methods. The `@digitalbazaar/http-client` already contains error handling https://github.com/digitalbazaar/http-client/blob/master/main.js#L41-L62.

Reply:
Removed here 47f0a4c.